lib/turn.rb

```diff
@@ -0,0 +1,34 @@
+class Turn
+  GAME_RULES = {
+    rock: {rock: :draw, paper: :lose, scissors: :win},
+    paper: {rock: :win, paper: :draw, scissors: :lose},
+    scissors: {rock: :lose, paper: :win, scissors: :draw}
+  }
+  attr_reader :name, :player_option, :computer_option
+
+  def initialize(options)
+    @name = options["name"]
+    @player_option = options["player_option"]
+    @computer_option = options["computer_option"]
+  end
+
+  def win?
+    result == :win
+  end
+
+  def lose?
+    result == :lose
+  end
+
+  def draw?
+    result == :draw
+  end
```

Comment:
This could maybe be cut down a bit, not sure if it would be less clear though so maybe it's better to leave it.